service/lib/agama/proxy_setup.rb

```diff
@@ -48,20 +48,25 @@ def initialize
       Yast.import "Proxy"
       Yast.import "Installation"
       Yast.import "PackagesProposal"
+
+      Proxy.Read
     end
 
     def run
       read
       write
     end
 
+    def propose
+      on_target { add_packages } if Proxy.enabled
```

Comment:
No need to call `on_target`. If this code runs as part of the manager service, it uses D-Bus to talk to the software service, which takes care of installing that package later. Or am I missing something?

Reply:
You are right, the problem is that I'm calling it after software proposal, but fixed in next commit.